src/NexusMods.Games.RedEngine/Cyberpunk2077/Diagnostics.cs

```diff
@@ -142,13 +175,22 @@ the following usecase of {DependencyName}
     internal static IDiagnosticTemplate MissingProtontricksForRedMod = DiagnosticTemplateBuilder
         .Start()
         .WithId(new DiagnosticId(Source, number: 3))
-        .WithTitle("Missing 'Protontricks' dependency")
+        .WithTitle("Protontricks is not installed")
         .WithSeverity(DiagnosticSeverity.Critical)
-        .WithSummary("Protontricks is required to install REDmods but is not present.")
+        .WithSummary("Protontricks is required to use REDmods but is not present.")
         .WithDetails("""
-Protontricks is required to install REDmods but is not present.
+Protontricks is not installed by is required to use the REDmod DLC when playing Cyberpunk 2077 on Linux.
+
+### How to Resolve
+1. Install Protontricks by following the instructions in the {ProtontricksUri}.
+2. Apply a loadout with Protontricks installed.
```

Comment:
I think we might need the user to restart the app for this to detect the presence of protontricks, but I'm not 100% on that

Reply:
@Sewer56 do you happen to know? The command to check for PT seems to run every time the Health Check is evaluated.